src/tigris.ts

```diff
@@ -206,6 +207,35 @@ export class Tigris {
 		defaultMetadata.set(USER_AGENT_KEY, USER_AGENT_VAL);
 		defaultMetadata.set(DEST_NAME_KEY, config.serverUrl);
 
+		// TODO: expose retry config to end user once validated
+		const svcConfig: ServiceConfig = {
+			loadBalancingConfig: [],
+			methodConfig: [
+				{
+					name: [
+						{
+							service: "tigrisdata.v1.Tigris",
+						},
+						{
+							service: "tigrisdata.search.v1.Search",
+						},
+					],
+					waitForReady: true,
+					retryPolicy: {
+						maxAttempts: 3,
+						initialBackoff: "0.1s",
+						maxBackoff: "1.0s",
+						backoffMultiplier: 1.5,
+						retryableStatusCodes: [Status.UNAVAILABLE, Status.UNKNOWN],
```

Comment:
Is it ok to retry on UNKNOWN?

Reply:
we need to retry on RESOURCE_EXHAUSTED after backoff.

Reply:
We also need to retry on:
- INTERNAL
- BAD_GATEWAY
- RESOURCE_EXHAUSTED

Reply:
Added, Bad gateway maps to INTERNAL grpc status.